Mastodon/Scene/Settings/General Settings/GeneralSettingSelectionCell.swift

```diff
@@ -45,5 +50,17 @@ class GeneralSettingSelectionCell: UITableViewCell {
 
         contentConfiguration = content
     }
+    
+    private func configureLanguageSetting(_ setting: GeneralSetting.Language, viewModel: GeneralSettingsViewModel) {
+        tintColor = Asset.Colors.Brand.blurple.color
+        accessoryType = .none
+        
+        var content = defaultContentConfiguration()
+        content.text = setting.title
+        content.secondaryText = LanguagePicker.availableLanguages().first(where: { $0.localeId == UserDefaults.shared.defaultPostLanguage })?.exonym
```

Comment:
The smaller text looked weird, so I checked Figma. According to Figma, it should look like this:

![Screenshot 2024-02-28 at 09 08 05](https://github.com/mastodon/mastodon-ios/assets/2580019/cd2c0c81-65a2-42e0-af43-bebe9790870c)

Currently it looks like this:

![Screenshot 2024-02-28 at 09 08 13](https://github.com/mastodon/mastodon-ios/assets/2580019/02481fae-d552-4a69-b003-703dddb8580d)

Maybe you can have another look? IIRC it's somethingsomething `UIListContentConfiguration.valueCell()` (but I'm not sure TBH